tests/ptvsd/server/test_attach.py

```diff
@@ -44,18 +44,16 @@ def test_attach(run_as, wait_for_attach, is_attached, break_into):
 
         if break_into == "break":
             assert session.read_json() == "break_into_debugger"
-            hit = session.wait_for_thread_stopped()
-            frames = hit.stacktrace.body["stackFrames"]
-            assert 32 == frames[0]["line"]
+            hit = session.wait_for_stop()
+            assert 32 == hit.frames[0]["line"]
```

Comment:
Can you switch this one to use `#@bp` also? It should work in named files just the same, but you'll have to call `tests.code.get_marked_line_numbers` on it explicitly.

Reply:
Missed this one. I had changed others in a previous PR